lib/blue_state_digital/error_middleware.rb

```diff
@@ -11,7 +12,9 @@ def on_complete(env)
       when 403
         raise BlueStateDigital::Unauthorized, response_values(env).to_s
       when 409
-        if env.body =~ /does not exist/
+        if env.body =~ /<error>/
+          raise BlueStateDigital::XmlErrorResponse, env.body
```

Comment:
Well, in the case of `process_signup`, we parse the XML error in that call so we can extract the field names from their IDs.  I'm not sure if there's a generalizable case for parsing the XML here.